module.yaml

```diff
@@ -28,9 +28,13 @@ spec:
         - source:
             protocol: s3
             dataformat: csv
+        - source:
+            protocol: fybrik-arrow-flight
```

Comment:
Does this source allow the chaining modules?

Reply:
yes. It allows the arrow-flight-module to talk to other modules that support the fybrik-arrow-flight protocol